### DISCUSSION TITLE
How to show a Persian calendar?

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [zauberzeug/nicegui#430] [Q&A] [answered]
Here in Qasar we can have [persian calendar](https://quasar.dev/vue-components/date#persian-calendar).
I am not sure how can I have it in NiceGUI.
Thanks for info.

**Comment by falkoschindler** [comment, marked as answer]
You can switch to a Persian calendar using the "calendar" prop. The value format is automatically "YYYY/MM/DD".

```py
ui.date(value='2023-02-24')
ui.date(value='1401/12/05').props('calendar=persian')
```

However, this does not translate anything. Language packs and internationalization with NiceGUI is still an open topic (#389).

**Comment by Shahin-rmz** [comment]
I want it to make some documents in Persian. 
I can translate the numbers to Persian one later, using other python packages.
But for now, this works for me.
Looking for `locale` enhancement in the future. 
Thanks
